recipes-utils/gpio-utils/files/uz7ev-evcc-hdmi-v/gpio_map.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Should be on  recipes-utils/gpio-utils/files/**uz7ev-evcc-hdmi**/gpio_map.sh instead of recipes-utils/gpio-utils/files/**uz7ev-evcc-hdmi-v**/gpio_map.sh

Reply:
I needed this to be fixed, so I took the 2 commits from this PR, and created a new one:
https://github.com/Avnet/meta-avnet/pull/49
So, we can discard this PR